src/Tickets/Commerce/Gateways/PayPal/Merchant.php

```diff
@@ -912,6 +912,27 @@ public function get_locale() {
 		 *
 		 * @param string $locale
 		 */
-		return apply_filters( 'tec_tickets_commerce_gateway_paypal_merchant_locale', $locale );
+		$locale = apply_filters( 'tec_tickets_commerce_gateway_paypal_merchant_locale', $locale );
+
+		return $this->validate_locale( $locale );
+	}
+
+	/**
+	 * Validates the locale to make sure it has a valid length.
+	 * If it's not, it will return the default locale.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $locale The Locale code.
+	 *
+	 * @return string
+	 */
+	public function validate_locale( string $locale ) : string {
+		// check if the length is valid.
+		if ( 5 !== strlen( $locale ) ) {
+			return 'en_US';
```

Comment:
Should we store the default locale in a class var and use it from there? It'd be nice to make this value filterable

Reply:
I agree. I think the default should be a variable so that we could read it from outside this class, if needed.